README.md

```diff
@@ -1,8 +1,7 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Mobile Version
-
-[Screenshot] <img src="Images/Screenshot4.png" width="300" height="2400">
+ <img src="Images/Screenshot5.png" width="1000" height="1000">
```

Comment:
- [ ] [OPTIONAL] Change this image due to it doesn't look like a good quality image, actually, you can only show one part you want to highlight about your project 😉

Reply:
Well solved